main.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 type config struct {
-	Token    string `env:"GITHUB_TOKEN" long:"token" description:"GitHub access token" required:"true"`
-	OrgName  string `env:"GITHUB_ORG" long:"org" description:"GitHub organization name" required:"true"`
-	Template string `env:"TEMPLATE" long:"template" description:"Go template (optional)" default:""`
-	Output   string `env:"OUTPUT" long:"output" description:"Output file" default:"output/graph.dot"`
+	HideTeamMembers bool   `env:"HIDE_TEAM_MEMBERS" long:"show_team_members" description:"Show Team Members on the diagram" required:"true"`
+	Token           string `env:"GITHUB_TOKEN" long:"token" description:"GitHub access token" required:"true"`
+	OrgName         string `env:"GITHUB_ORG" long:"org" description:"GitHub organization name" required:"true"`
+	Template        string `env:"TEMPLATE" long:"template" description:"Go template (optional)" default:""`
+	Output          string `env:"OUTPUT" long:"output" description:"Output file" default:"output/graph.dot"`
```

Comment:
As few suggestions:

- this new config is not so important to be the first in the list, I think it should be under `OrgName` or `Output`;
- project name already has "team" in it, so it can be omitted from parameter name;
- let's keep it optional, it will be `false` by default anyway;
- underscores for CLI flags are not common, [GNU argument convention](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) tells
  > Option names are typically one to three words long, with hyphens to separate words

```suggestion
	Token       string `env:"GITHUB_TOKEN" long:"token" description:"GitHub access token" required:"true"`
	OrgName     string `env:"GITHUB_ORG" long:"org" description:"GitHub organization name" required:"true"`
	HideMembers bool   `env:"HIDE_MEMBERS" long:"hide-members" description:"Hide Team Members on the diagram"`
	Template    string `env:"TEMPLATE" long:"template" description:"Go template (optional)" default:""`
	Output      string `env:"OUTPUT" long:"output" description:"Output file" default:"output/graph.dot"`
```